development/metamaskbot-build-announce.js

```diff
@@ -100,6 +100,7 @@ async function start() {
   const moduleInitStatsUILink = `<a href="${moduleInitStatsUIUrl}">MV3 UI Init Stats</a>`;
   const moduleLoadStatsUrl = `${BUILD_LINK_BASE}/test-artifacts/chrome/mv3/load_time/index.html`;
   const moduleLoadStatsLink = `<a href="${moduleLoadStatsUrl}">Module Load Stats</a>`;
+  const e2eActionsStatsLink = `${BUILD_LINK_BASE}/test-artifacts/chrome/mv3/user_actions/stats.json`;
```

Comment:
Hey @seaona : can you do something like this:
```
  const moduleLoadStatsUrl = `${BUILD_LINK_BASE}/test-artifacts/chrome/mv3/load_time/index.html`;
  const moduleLoadStatsLink = `<a href="${moduleLoadStatsUrl}">Module Load Stats</a>`;

```

Reply:
thank you! added in the next commit @jpuri